decidim-core/app/services/decidim/resource_search.rb

```diff
@@ -39,12 +39,21 @@ def search_category_id
     # `"global"` as parameter, and in the method we do the needed changes to search
     # properly.
     #
-    # You can use the `search_organization_scopes` helper method, defined in
-    # `Decidim::OrganizationScopesHelper`, to render the collection needed for the
+    # You can use the `search_participatory_process_subscopes` helper method, defined in
+    # `Decidim::ParticipatoryProcessScopesHelper`, to render the collection needed for the
     # `collection_check_boxes` form method.
     def search_scope_id
-      clean_scope_ids = [scope_id].flatten.map { |id| id == "global" ? nil : id }
-      query.where(decidim_scope_id: clean_scope_ids)
+      clean_scope_ids = [scope_id].flatten
+
+      conditions = []
+      conditions << "decidim_scope_id IS NULL" if clean_scope_ids.delete("global")
```

Comment:
Mmm, I'm not doing the join if the only condition is "global", that is why I use the feature column instead of the scope's column.